lib/statusbar-summary-view.coffee

```diff
@@ -36,17 +34,18 @@ class StatusBarSummary extends View
 
   @content: (messages, editor, info, warning, error) ->
     @div class: 'linter-summary inline-block', =>
-      if error > 0
-        @div class: 'linter-error inline-block', error, =>
-          @span class: 'icon-right'
-      if warning > 0
-        @div class: 'linter-warning inline-block', warning, =>
+      if info + warning + error == 0
+        @div class: 'linter-success inline-block', =>
           @span class: 'icon-right'
-      if info > 0
-        @div class: 'linter-info inline-block', info, =>
-          @span class: 'icon-right'
-
-  detached: ->
-    @off 'click', '.linter-summary-click-container'
```

Comment:
Useless line (I think I am the one to blame :imp: ), we are listening on:
```
    @on 'click', ->
      moveToNextMessage messages, editor
```
I am not sure but I think it's useless to remove the listener manually because it is destroyed by this line: https://github.com/ababol/Linter/blob/feature-statusbar-sum-success-dot/lib/statusbar-summary-view.coffee#L8 , nop?